ironfish/src/memPool/memPool.ts

```diff
@@ -20,6 +27,11 @@ export class MemPool {
   constructor(options: { strategy: Strategy; chain: Blockchain; logger?: Logger }) {
     const logger = options.logger || createRootLogger()
 
+    this.queue = new FastPriorityQueue<TransactionFeeAndHash>(
+      (firstTransaction, secondTransaction) => {
+        return firstTransaction.fee > secondTransaction.fee
```

Comment:
For this comparator what happens if two fees are equal, is this stable sort? Should we have a second ordering by hash?